fix(html): Add the qualifier to global attribute page titles

dipikabh opened the pull request:
### Description

We have the qualifier "HTML attribute:" on attribute pages ([example](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/accept)), but it is missing from titles for global attributes. 
This PR adds "HTML global attribute:" to the `title` key and repurposes the existing title as `short-title`.

### Motivation

To ensure consistency in title format in the same technology area. This also benefits SEO.

dipikabh committed:
update titles

chrisdavidmills reviewed (approved):
LGTM! Nice change, @dipikabh, this will improve human comprehension as well as SEO.

chrisdavidmills commented:
> LGTM! Nice change, @dipikabh, this will improve human comprehension as well as SEO.

@dipikabh oops, I've just noticed that this is a draft. Let me know when you are ready for me to take another look.

dipikabh committed:
update titles to drop html

dipikabh commented:
Sorry @chrisdavidmills - I hit Draft as fast as I could so you wouldn't have to review again.
You were too quick! :)

Essentially, I put the PR on hold to push out a different change. @bsmth pointed out (and I agree) that the updated front matter makes the meta `<title>` on the page to look like:

`<title>HTML global attribute: exportparts - HTML: HyperText Markup Language | MDN</title>`

That is what's happening for attribute pages, like for https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/accept, it is:

`<title>HTML attribute: accept - HTML: HyperText Markup Language | MDN</title>`

To avoid that "HTML...HTML" repetition, I've dropped the technology prefix. In this new commit, I've updated the titles to follow this pattern:

```diff
- title: HTML global attribute: exportparts
+ title: exportparts global attribute
```

This now renders as:

`<title>exportparts global attribute - HTML: HyperText Markup Language | MDN</title>`

This style is slightly different from our existing attribute and element titles, but we're going to watch out for any negative SEO before making changes on other pages.

This is ready for your review now, thank you!

chrisdavidmills reviewed (approved):
@dipikabh this is a bit of a shame, as it makes the actual page headings a bit less strong, but I can understand and appreciate the compromise you've made here.

I think it still makes the headings better.

bsmth reviewed (approved):
Nice 👍🏻 All looks good to me

dipikabh committed:
add the tech back
